.github/labeler.yml

```
@@ -0,0 +1,5 @@
+feature-definition:
```

Anticipating that we'll rename feature-group-definitions/ to just features/:

```suggestion
features:
```

---

This is the label name, not the path—I've put in cc1be44 to make sure we cover that from the get go (I'm anticipating a second label for dist-only changes, so I'd like to keep the label name as-is).